app/assets/stylesheets/modules/spotlight_overrides.scss

```diff
@@ -309,3 +309,12 @@ h5,
     z-index: 1;
   }
 }
+
+// Overrides for search tips link and info icon
+.image-masthead .searchtips-link {
+  text-transform: none !important;
+}
+
+.searchtips-link .bi-info-circle{
+  margin-bottom: 0.15rem;
+}
```

Comment:
Are these styles needed? The link text looks the same if I remove them and the icon position seems okay.

Reply:
The first style applies when we add an image to the masthead.  When that happens, there is an existing style that changes the text to uppercase.  The style on line 314 prevents that from happening for search tips, since these should stay lowercase even with an image masthead.

For the second style, the vertical spacing did not seem exactly where the Figma screen had it.

Reply:
Okay. It's really strange that spotlight upcases the nav links when there's background image, but otherwise doesn't.